packages/cli/src/commands/db.ts

```diff
@@ -1,13 +1,17 @@
-import {spawn} from 'cross-spawn'
-import {flags} from '@oclif/command'
-import {Command} from '@oclif/command'
+import {resolveBinAsync} from '@blitzjs/server'
+import {Command, flags} from '@oclif/command'
 import chalk from 'chalk'
+import {spawn} from 'cross-spawn'
+import {prompt} from 'enquirer'
+import * as fs from 'fs'
 import * as path from 'path'
-import {resolveBinAsync} from '@blitzjs/server'
+import {promisify} from 'util'
+import {loadBlitz} from '../utils/load-blitz'
 
 const schemaPath = path.join(process.cwd(), 'db', 'schema.prisma')
 const schemaArg = `--schema=${schemaPath}`
 const getPrismaBin = () => resolveBinAsync('@prisma/cli', 'prisma')
+const {db} = loadBlitz()
```

Comment:
```suggestion
const db = require(path.join(process.cwd(), 'db').default
```
I didn't actually test this, but I think this is correct

Reply:
I was doing it like this first but it says `Error Plugin: @blitzjs/cli: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.` Without default I was able to get the database connections but raw wasn't working.

Reply:
Okay it works. Had to run migrate again forgot about that with all the swapping between databases. But I can't get the test to pass with this import it says `'Cannot find module '/home/lori/dev/oss/blitz/blitz/packages/cli/db' from 'db.ts'' from 'db.ts'`. It searches for this file at the cli package root. Do I have to add something to the tests?

Reply:
Ah yeah, you need to do the same thing that `loadBlitz` is doing.
```ts
import pkgDir from 'pkg-dir'
const projectRoot = pkgDir.sync() || process.cwd()
```

You can move that into a utils file or something if you want.